pkg/logs/client/http/destination.go

```diff
@@ -271,9 +271,11 @@ func (d *Destination) unconditionalSend(payload *message.Payload) (err error) {
 		req.Header.Set("DD-EVP-ORIGIN", string(d.origin))
 		req.Header.Set("DD-EVP-ORIGIN-VERSION", version.AgentVersion)
 	}
-	req = req.WithContext(ctx)
-
+	req.Header.Set("dd-message-timestamp", strconv.FormatInt(getMessageTimestamp(payload.Messages), 10))
```

Comment:
Should this be `DD-EVP-MESSAGE-TIMESTAMP` to be consistent with the other headers?

Reply:
We have argued about this internally, and in fact it was my first proposal.

The result was that as "evp" stands for "event-platform", it is not really relevant for those headers.